docs/whatsnew_1.5.7.html

```diff
@@ -9,10 +9,9 @@
      */
 //-->
 <head>
-    <title>What's New in Zen Cart&reg; v1.5.7</title>
     <meta charset="utf-8">
+    <title>What's New in Zen Cart&reg; v1.5.7</title>
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <meta http-equiv="Content-Type" content="text/html">
```

Comment:
Why this change?

Reply:
This replaces it
`<meta charset="utf-8">`
and you can't have both.

Reply:
ok